### PR TITLE
support $(this-folder)

### PR DESCRIPTION
**Pull request by ruowan**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -282,6 +282,7 @@ const getInputFilesFromReadme = (readMePath: string): asyncIt.AsyncIterableEx<Sp
 
     yield* openApiMd
       .getInputFiles(m.markDown)
+      .map(f => f.replace('$(this-folder)', '.'))
```
Comment (raych1) [inline src/index.ts]: Does it search from current folder by default without explicitly write it?

File: src/test/parse_this_folder/specification/readme.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+```
+
+---
+
+```yaml $(tag) == 'all-api-versions'
+require: $(this-folder)/../../../../profiles/readme.md
+
+# all the input files across all versions
+input-file:
+  - $(this-folder)/specs/b.json
```
Comment (raych1) [inline src/test/parse_this_folder/specification/readme.md]: Prefer to call it `current-folder` for better understanding.

Comment (ruowan) [reply src/test/parse_this_folder/specification/readme.md]: Because in swagger readme.md file. it called 'this-folder'.

File: src/index.ts (hunk 1 above)
Comment (ruowan) [reply src/index.ts]: Yes. It searches from the readme file parent directory.  So the path here is correct.

**Pull request merged**